src/toggle/Toggle.tsx

```diff
@@ -0,0 +1,48 @@
+import "./_toggle.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+import ToggleItem from "./item/ToggleItem";
+import {ToggleContext} from "./util/ToggleContext";
+
+export interface ToggleProps {
+  children: React.ReactNode;
+  onToggle: (dataId: string) => void;
```

Comment:
Not sure about this but I think we should handle this within Toggle and pass the newly generated "selectedItems" array to the parent. What do you think ?
```suggestion
  onChange: (selectedItems: string[]) => void;
```

This way we can move the following function to Toggle and handle the manipulation within Toggle component itself:

```
function handleToggle(dataId: string, state: any) {
  let selectedItems;

  if (state.includes(dataId)) {
    selectedItems = state.filter((item) => item !== dataId);
  } else {
    selectedItems = [...state, dataId];
  }

  return selectedItems;
}
```

cc: @bilgeyucel @mucahit @jamcry @Anlerkan

Reply:
In this case, we need `isMultiple` prop. Because the user might want to select only one item. If `isMultiple` is false, we can return just one item.

Reply:
I think Ediz's suggestion makes sense. Currently, we are already keeping array for each state
```
  switch: [],
  devices: [],
  frameworks: []
```
We can add something like `canSelectMultiple`. I think it was named like that in `TypeaheadSelect`.